### PR TITLE
Upgrade react to 0.13.x

### PR DESCRIPTION
**Pull request by mridgway**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by mridgway** [comment]
Looks like node v0.10 ships with a version of npm that doesn't like our React dependency range. Will have to investigate.

**Comment by mridgway** [comment]
Fixed by upgrading to new flux-router-component and fluxible.

File: services/docs.js [hunk 1]
```diff
@@ -56,6 +57,7 @@ walker.on('end', function () {
 module.exports = {
     name: 'docs',
     read: function (req, resource, params, config, callback) {
+        console.log(params.key, Object.keys(content));
```
Comment (redonkulus) [inline services/docs.js]: Remove

File: Gruntfile.js [hunk 2]
```diff
@@ -95,7 +95,7 @@ module.exports = function (grunt) {
                 module: {
                     loaders: [
                         { test: /\.css$/, loader: 'style!css' },
-                        { test: /\.jsx$/, loader: 'jsx-loader' },
+                        { test: /\.(js|jsx)$/, exclude: /node_modules/, loader: require.resolve('babel-loader') },
```
Comment (jedireza) [inline Gruntfile.js]: Regex could be simiplified to

```js
/\.jsx?$/
```

**Comment by mridgway** [comment]
This should be ready to go.

**Comment by jedireza** [comment]
:+1: :ship:

**Pull request merged**